### ISSUE TITLE
klayout pull access denied

### ISSUE THREAD

**Issue by p-brane** [efabless/foss-asic-tools#6]
I'm running Kubuntu 22.04 on VirutualBox 6.1.40 on a Windows 10 22H2 host machine. I install docker on Kubuntu and then ran the foss-asic-tools tools install. Klayout failed to install (see below) What's the best way to install it? 

`[ERROR 17-Dec-22 05:06:33] pull access denied for klayout, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
Image |              base              | Success    
Image |             magic              | Success    
Image |          pdks-volare           | Success    
Image |            openlane            | Success    
Image |            klayout             | Fail       
Image |            gtkwave             | Success    
Image |            iverilog            | Success    
Image |             netgen             | Success    
Image |   riscv-gnu-toolchain-rv32i    | Success    
Image |          gaw3-xschem           | Success    
Image |            ngscope             | Success    
Image |            ngspice             | Success    
Image |             xschem             | Success    
Image |             kicad              | Success    
Image |              xyce              | Success    
Image |            covered             | Success    
Image |             irsim              | Success    
Image |            opensta             | Success    
Image |              cvc               | Success    
Image |            iic-osic            | Success    
Image |        foss-asic-tools         | Fail  `

**Comment by p-brane** [comment]
I started installing the packages manually using [Open Toolchain Installation](https://anthonywall.ie/opentools/) on Kubuntu 22.04 running in on Viraualbox 6.1.40 on the Windows 10 22H2 host. I found that the klayout fails because qt5-defaults has been depreciated on Kubuntu/Ubuntu 22.04. The error reported is `E: Package 'qt5-default' has no installation candidate`. I installed `sudo apt-get install qtbase5-dev qtchooser qt5-qmake qtbase5-dev-tools` and then `sudo apt-get install ruby ruby-dev`. 

The klayout installer then reported that `*** ERROR: wrong -pylib path: /opt/anaconda3/lib/x86_64-linux-gnu/libpython3.9.a does not exist or not a file```` I created a conda environment to use python 3.10 and linked `ln -s /usr/lib/x86_64-linux-gnu/ /opt/anaconda3/envs/py310/lib/` the python library to the anaconda directory.

Perhaps, something like this is happening during the foss-asic-tools docker install on Ubuntu 22.04.

**Comment by p-brane** [comment]
I started installing the packages manually using [Open Toolchain Installation](https://anthonywall.ie/opentools/) on Kubuntu 22.04 running in on Viraualbox 6.1.40 on the Windows 10 22H2 host. I found that the klayout fails because qt5-defaults has been depreciated on Kubuntu/Ubuntu 22.04. The error reported is `E: Package 'qt5-default' has no installation candidate`. I installed `sudo apt-get install qtbase5-dev qtchooser qt5-qmake qtbase5-dev-tools` and then `sudo apt-get install ruby ruby-dev`. 

The klayout installer then reported that `*** ERROR: wrong -pylib path: /opt/anaconda3/lib/x86_64-linux-gnu/libpython3.9.a does not exist or not a file```` I created a conda environment to use python 3.10 and linked `ln -s /usr/lib/x86_64-linux-gnu/ /opt/anaconda3/envs/py310/lib/` the python library to the anaconda directory.

Perhaps, something like this is happening during the foss-asic-tools docker install on Ubuntu 22.04.
